cypress/e2e/nonParallel/data-import/e2e/mrc-import-file-for-create-items.cy.js

```diff
@@ -88,7 +88,7 @@ describe('ui-data-import', () => {
     NewFieldMappingProfile.fillSummaryInMappingProfile(itemMappingProfile);
     NewFieldMappingProfile.fillMaterialType('"book"');
     NewFieldMappingProfile.fillPermanentLoanType('"Can circulate"');
-    NewFieldMappingProfile.fillStatus('"Available"');
+    NewFieldMappingProfile.fillStatus('Available');
```

Comment:
use ITEM_STATUSES.AVAILABLE, LOAN_TYPE_NAMES.CAN_CIRCULATE, MATERIAL_TYPE_NAMES.BOOK

Reply:
the same